File: PooCC01CNetz/Semana05/Python/Ejercicios01/Date.py
class Date:
    def __init__(self, day = None, mounth = None, year = None):
        if day is None and mounth is None and year is None:
            self.__day = None
            self.__mounth = None
            self.__year = None
        else:
            self.__day = day
            self.__mounth = mounth
            self.__year = year
    
    def setDay(self, day):
        self.__day = day

    def getDay(self):
        return self.__day
    
    def setMounth(self, mounth):
        self.__mounth = mounth

    def getMounth(self):
        return self.__mounth
    
    def setYear(self, year):
        self.__year = year

    def getYear(self):
        return self.__year

File: PooCC01CNetz/Semana05/Java/Ejercicios01/Date.java
public class Date {
    private int day;
    private int mounth;
    private int year;
    
    Date(int day, int mounth, int year){
        this.day = day;
        this.mounth = mounth;
        this.year = year;
    }

    Date() {}

    public void setDay(int day) {
        this.day = day;
    }

    public int getDay() {
        return day;
    }

    public void setMounth(int mounth) {
        this.mounth = mounth;
    }

    public int getMounth() {
        return mounth;
    }

    public void setYear(int year) {
        this.year = year;
    }

    public int getYear() {
        return year;
    }
}
